dark/diamond/alignments.py

```diff
@@ -0,0 +1,199 @@
+from random import uniform
+from math import log10
+import copy
+
+from dark.score import HigherIsBetterScore
+from dark.alignments import ReadsAlignments, ReadsAlignmentsParams
+from dark.blast.conversion import JSONRecordsReader
+from dark.blast.params import checkCompatibleParams
+from dark import ncbidb
+from dark.utils import numericallySortFilenames
+
+ZERO_EVALUE_UPPER_RANDOM_INCREMENT = 150
```

Comment:
Or maybe not. I don't know if diamond does the same things as blast. I guess we can just leave this in, just in case.